rmw_connext_xtypes_dynamic_cpp/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ find_package(rosidl_generator_cpp REQUIRED)
 
 find_package(rosidl_typesupport_introspection_cpp REQUIRED)
 ament_export_dependencies(rosidl_typesupport_introspection_cpp)
-
+ament_export_include_directories(include/${PROJECT_NAME}/impl ${Connext_INCLUDE_DIRS})
```

Comment:
I would suggest moving the include dir export either above or below both dependency exports.